### PR TITLE
Create comagic.json

### PR DESCRIPTION
**Pull request by dred-art**

### CODE REVIEW

File: sources/comagic.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  " id " : " COMAGIC_DATAAPI " ,
```
Comment (pfrisella) [inline sources/comagic.json]: This should match filename: COMAGIC

File: sources/comagic.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  " name " : " CoMagic " ,
+  " category " : [ " АНАЛИТИКА " , " CALL_TRACKING ", " DATA_COLLECTION ", " РЕКЛАМА " ],
```
Comment (pfrisella) [inline sources/comagic.json]: we don't yet support languages other than english. please fix.

File: sources/comagic.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  " организация " : " COMAGIC " ,
```
Comment (pfrisella) [inline sources/comagic.json]: organizations

File: sources/comagic.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/comagic.json]: remove leading spaces from all entries

Comment (pfrisella) [reply sources/comagic.json]: and in general extra whitespace should be removed

**Comment by dred-art** [comment]
Thank. Made a change

**Pull request merged**